options.go

```diff
@@ -277,7 +277,8 @@ func hasArg(s string) bool {
 		s = "-" + s
 	}
 	for _, arg := range os.Args {
-		if strings.Split(arg, "=")[0] == s {
+		key := strings.Split(arg, "=")[0]
+		if key == s || key == "-"+s {
```

Comment:
would it be better to compare the non prefixed versions of both the input to this function and the args we're iterating over?